isb_web/main.py

```diff
@@ -262,7 +265,10 @@ def thing_list(
     total_records, npages, things = sqlmodel_database.read_things_summary(
         session, offset, limit, status, authority
     )
-
+    properties = {
+        "authority": authority or "None"
+    }
```

Comment:
These custom properties allow us to segment the analytics events and get counts that way on the dashboard, e.g.

<img width="1079" alt="image" src="https://user-images.githubusercontent.com/1188978/149235833-92639f7b-bad3-48c6-96ab-27c037867806.png">